extension/natives/cbasenpc.h

```diff
@@ -231,13 +231,14 @@ CBASENPCLOCONATIVE(SetCallback)
 CBASENPCLOCONATIVE(CallBaseFunction)
 	if (!loco->IsInCallback())
 	{
-		pContext->ThrowNativeError("CallBaseFunction() can only be used within a callback");
+		pContext->ReportError("CallBaseFunction() can only be used within a callback");
```

Comment:
Any specific reason for this change, this seems like we would still want to abort plugin stack if they did such error. If this connected to the crash mentioned, it would be preferrable to make it just not happen.

Reply:
After digging up SM docs, looks like the function was deprecated. This change is fine.